gossipsub.go

```diff
@@ -287,6 +287,8 @@ func (gs *GossipSubRouter) Attach(p *PubSub) {
 	// connect to direct peers
 	if len(gs.direct) > 0 {
 		go func() {
+			// add a small delay to make this unit-testable
+			time.Sleep(time.Second)
```

Comment:
Should we make this a variable, so that we only add the delay when testing?

Reply:
Maybe so, but a 1s delay to connect can't hurt the real use case :)

Reply:
alright, I'll turn it into a variable -- not sure we gain much by it though, I'll leave it to its default value of 1s.

Reply:
done (will push shortly).